### DISCUSSION TITLE
Release versions of ImageSharp.Drawing and Fonts

### DISCUSSION THREAD

**Discussion by chris-steema** [SixLabors/ImageSharp.Drawing#277] [General] [answered]
Hello,

I notice that there are two milestones for v1.0.0 versions of both [ImageSharp.Drawing](https://github.com/SixLabors/ImageSharp.Drawing/milestone/3) and [Fonts](https://github.com/SixLabors/Fonts/milestone/2)—does this mean that release versions for these two projects are going to made public in the near future?

Thank you.

**Comment by JimBobSquarePants** [comment, marked as answer]
It does yes! 

I've been working very hard to bring both these libraries up to a level where I'm confident they can be released. As you can imagine though doing so is a lot of work so any assistance that can be offered would be most useful, even if it's just testing out the latest nightlies and providing feedback.

**Reply by chris-steema** [reply]
I'm afraid I'm not in a position to commit to helping regularly, but I would like to contribute in some way—do you have the time to run the unit tests in this repo on every nightly, for example?

**Comment by JimBobSquarePants** [comment]
Now! 😁
